### PR TITLE
feat: add Tested By Contributor custom field per test case

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Added "tested_by_contributor" custom field per test case in markdown or "Tested By Contributor" in Zephyr Scale as an option to record testing done by the community. This is in support for the upcoming Hacktoberfest where QA contributors may submit PR in this repo to indicate their testing/verification contribution in the corresponding PR in the main repo (ex. at https://github.com/mattermost/mattermost)

### CODE REVIEW

File: data/TEST_CASE_TEMPLATE.md
Comment (lindy65) [inline data/TEST_CASE_TEMPLATE.md:1]: Hi Saturn, should line 17 read "tested_by_contributor" (not "community")?

Comment (saturninoabril) [reply data/TEST_CASE_TEMPLATE.md:1]: Good catch Lindy, should be fixed now. Last time, I had a last minute fix and I totally missed this one. Thanks for thorough review!

Comment (lindy65) [reply data/TEST_CASE_TEMPLATE.md:1]: You're welcome Saturn!

**Review by lindy65 — APPROVED** [review]
Thanks @saturninoabril - looks good to me :+1:

**Pull request merged**